src/components/vertical-menu/responsive-vertical-menu/responsive-vertical-menu.component.tsx

```diff
@@ -103,6 +105,11 @@ const BaseMenu = ({
     }
   }, [active, menu, responsiveMode, activeMenuItem, button]);
 
+  useEffect(() => {
```

Comment:
comment: this can be removed and just pass the `height` and `width` directly as it's updated via props